plugins/source/awspricing/Makefile

```
@@ -0,0 +1,16 @@
+.PHONY: test
+test:
+	go test -timeout 3m ./...
+
+.PHONY: lint
+lint:
+	@golangci-lint run --timeout 10m
+
+.PHONY: gen-docs
+gen-docs:
+	rm -rf ./docs/tables/*
+	go run main.go doc ./docs/tables
```

```suggestion
	go run main.go doc ./docs/tables
	sed 's_(\(.*\))_(https://github.com/cloudquery/cloudquery/blob/main/plugins/source/awspricing/docs/tables/\1)_' docs/tables/README.md > ../../../website/pages/docs/plugins/sources/awspricing/tables.md
```

We need this for the Website (if we plan to add it to it)